C22_ROW_TYPE[] row
int32 xOffset
int32 yOffset
geometry_msgs/Point robotPos
geometry_msgs/Point robotOri
geometry_msgs/Point left_foot
geometry_msgs/Point right_foot
